ui/app/[locale]/home/ClustersTable.tsx

```diff
@@ -27,71 +40,105 @@ export function ClustersTable({
           case "name":
-            return <Th width={40}>Name</Th>;
+            return <Th width={25}>Name</Th>;
           case "nodes":
-            return <Th>Online Brokers</Th>;
-          case "producers":
-            return <Th>Producers</Th>;
+            return <Th>Brokers</Th>;
           case "consumers":
-            return <Th>Consumers</Th>;
+            return <Th>Consumer groups</Th>;
+          case "version":
+            return <Th>Kafka version</Th>;
+          case "namespace":
+            return <Th>Project</Th>;
         }
       }}
-      renderCell={({ column, row, Td }) => {
+      renderCell={({ key, column, row, Td }) => {
         switch (column) {
           case "name":
             return (
-              <Td>
+              <Td key={key}>
                 <Link href={`/kafka/${row.id}`}>
                   <Truncate content={row.attributes.name} />
                 </Link>
               </Td>
             );
           case "nodes":
             return (
-              <Td>
-                <Number value={row.attributes.nodes.length} />
-                /
-                <Number value={row.attributes.nodes.length} />
-              </Td>
-            );
-          case "producers":
-            return (
-              <Td>
-                <Number value={0} />
-                /
-                <Number value={0} />
+              <Td key={key}>
+                <Suspense fallback={<Skeleton />}>
+                  <NodesCell kafkaId={row.id} data={row.extra.nodes} />
+                </Suspense>
               </Td>
             );
           case "consumers":
             return (
-              <Td>
-                <Number value={0} />
-                /
-                <Number value={0} />
+              <Td key={key}>
+                <Suspense fallback={<Skeleton />}>
+                  <ConsumersCell
+                    kafkaId={row.id}
+                    data={row.extra.consumerGroupsCount}
+                  />
+                </Suspense>
               </Td>
             );
+          case "version":
+            return <Td key={key}>{row.attributes.kafkaVersion || "n/a"}</Td>;
+          case "namespace":
+            return <Td key={key}>{row.attributes.namespace}</Td>;
         }
       }}
       renderActions={({ ActionsColumn, row }) => (
         <ActionsColumn
           items={[
             {
-              title: "Connect to this cluster",
+              title: "Connection details",
               onClick: () => {
                 open(row.id);
               },
-              icon: <IntegrationIcon />,
             },
             {
-              title: "Copy Bootstrap URL",
-              onClick: () => {
-                navigator.clipboard.writeText(row.attributes.bootstrapServers);
-              },
-              icon: <CopyIcon />,
+              title: "View in OpenShift console",
+              icon: <ExternalLinkAltIcon />,
+              isDisabled: true,
```

Comment:
Do we want to display the OCP link while disabled?

Reply:
Can we make the link programmatically?

Reply:
We could probably read the `console` route in namespace `openshift-console` and append the path to read the Kafka CR. We should probably have an idea how we plan to handle Kubernetes (vanilla) also before we make that change though.

Reply:
Let's keep it disabled for the moment